runtime/src/accounts.rs

```diff
@@ -389,27 +405,22 @@ impl Accounts {
     pub fn transaction_count(&self) -> u64 {
         self.accounts_db.read().unwrap().transaction_count()
     }
-    ///// accounts starts with an empty data structure for every fork
-    ///// self is root, merge the fork into self
-    //pub fn merge_into_root(&self, other: Self) {
-    //    assert!(other.account_locks.lock().unwrap().is_empty());
```

Comment:
Is this lock no longer necessary? It seems like it might be ok b/c the accounts being touched by potential ongoing transactions should remain unchanged after a merge with the parent. Are there any other edge cases?

Reply:
sorry, I don't understand the question